buildspec.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023, OpenHW Group
+# SPDX-License-Identifier:Apache-2.0 WITH SHL-2.0
+#
+# Build-specification for the aws.yml workflow.
+
+version: 0.2
+
+phases:
+
+  build:
+    commands:
+      - chown -R florian:florian .
+#      - su florian -c "source /synopsys/scripts/env.sh; echo $PATH"
+# VCS environment
+      - yum -y install gcc gcc-c++ make
+      - export VCS_HOME=/synopsys/vcs/S-2021.09-SP1
+      - export PATH=$VCS_HOME/bin:$PATH
+      - vcs -id
+#      - export LM_LICENSE_FILE=29000@ip-172-31-46-244
+      - export SNPSLMD_LICENSE_FILE=27020@ip-172-31-46-244.eu-west-1.compute.internal
+# Questasim (vsim) environment
+#      - export QUESTAROOT=/onespin/questasim
+#      - export LM_LICENSE_FILE=29000@ip-172-31-46-244
+#      - export PATH=${QUESTAROOT}/bin:${PATH}
+#      - export MTI_VCO_MODE=64
+#      - wget https://buildbot.embecosm.com/job/corev-gcc-centos7/19/artifact/corev-openhw-gcc-centos7-20230905.tar.gz
```

Comment:
Maybe remove the commented lines?

Reply:
Yes, the commented lines will be removed before the Draft status of this PR is dropped.  These were left in place to show you what I did to get Questasim (vsim) to run.  It did not work.  Perhaps the value of `LM_LICENSE_FILE` needs to be set to the same value as we are using for `SNPSLMD_LICENSE_FILE`?  Any thoughts on this?